vault/data_source_auth_backend.go

```diff
@@ -77,15 +77,28 @@ func authBackendDataSourceRead(d *schema.ResourceData, meta interface{}) error {
 
 	path = strings.TrimSuffix(path, "/")
 	d.SetId(path)
-	d.Set("type", auth.Type)
-	d.Set("description", auth.Description)
-	d.Set("accessor", auth.Accessor)
-	d.Set("default_lease_ttl_seconds", auth.Config.DefaultLeaseTTL)
-	d.Set("max_lease_ttl_seconds", auth.Config.MaxLeaseTTL)
-	d.Set("listing_visibility", auth.Config.ListingVisibility)
-	d.Set("local", auth.Local)
-	return nil
 
-	// If we fell out here then we didn't find our Auth in the list.
+	if err := d.Set("type", auth.Type); err != nil {
```

Comment:
If we want the error checks for d.Set to be ignored, it seems like we can set up an exclusion: https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml#L266-L299